docs/api/index.rst

```diff
@@ -52,15 +55,21 @@ HTTP endpoints
 Many of the listed endpoints are resource endpoints, which an be filtered,
 paginated and interracted as described in :ref:`resource-endpoints`.
 
-Kinto protocol
+
+Full reference
 ==============
 
-In addition to the endpoints defined above, some aspects of the API might be
-of interest:
+Full detailed API documentation:
 
 .. toctree::
    :maxdepth: 2
 
+   buckets
```

Comment:
I put these at the end of the listing on purpose: the table already mentions all the endpoints which are here.